contracts/cw20-stake-external-rewards/src/contract.rs

```diff
@@ -99,14 +100,41 @@ pub fn instantiate(
         )
         .add_attribute("reward_rate", reward_config.reward_rate)
         .add_attribute("period_finish", reward_config.period_finish.to_string())
-        .add_attribute("reward_duration", reward_config.reward_duration.to_string()))
+        .add_attribute("reward_duration", reward_config.reward_duration.to_string())
+        .add_attribute("is_paused", "false"))
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    // Set contract to version to latest
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
-    Ok(Response::default())
+    match msg {
+        MigrateMsg::FromV1 { is_paused } => {
+            // Load the current config and introduce the claim pause flag
+            let current_config =
+                stake_cw20_external_rewards_v1::state::CONFIG.load(deps.storage)?;
+            let is_paused = if let Some(val) = is_paused {
```

Comment:
maybe we should just make `is_paused` be a `bool` instead of `Option<bool>`? not sure what we get by making `false` indicated by both `None` and `false`

Reply:
I was thinking about this one a bunch and had the same thought. Will do!